app/main.py

```diff
@@ -1 +1,54 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```

Comment:
Why do you want it to be class attribute? What if one engineer will learn some skill, does it mean other engineers also learned it? Doesn't look logical, you should fix it